portal/templates/portal_wrapper.html

```diff
@@ -144,7 +144,7 @@
         </div>
     </div>
 </div>
-<input type="hidden" id="sessionMonitorProps" data-baseurl="{{PORTAL}}" data-crsftoken="{{csrf_token()}}" data-sessionlifetime="{{config.PERMANENT_SESSION_LIFETIME if config.PERMANENT_SESSION_LIFETIME}}" />
+<input type="hidden" id="sessionMonitorProps" data-baseurl="{{PORTAL}}" data-crsftoken="{{csrf_token()}}" data-sessionlifetime="{{config.PERMANENT_SESSION_LIFETIME if config.PERMANENT_SESSION_LIFETIME}}" data-cookie-timeout="{{request.cookies.get('SS_TIMEOUT')}}"/>
```

Comment:
I might change those config lookups to fail if they're not configured, since those config values are required ie:
```
{{request.cookies['SS_TIMEOUT']}}
{{config.PERMANENT_SESSION_LIFETIME}}
```